code/modules/mob/living/carbon/carbon.dm

```diff
@@ -263,20 +263,28 @@ mob/living
 				AdjustWeakened(-3)
 				playsound(src.loc, 'sound/weapons/thudswoosh.ogg', 50, 1, -1)
 				if(!player_logged)
-					M.visible_message( \
-						"\blue [M] shakes [src] trying to wake [t_him] up!", \
-						"\blue You shake [src] trying to wake [t_him] up!", \
+					M.visible_message(\
+						"<span class ='notice'> [M] shakes [src] trying to wake [t_him] up!</span>",\
+						"<span class ='notice'> You shake [src] trying to wake [t_him] up!</span>",\
```

Comment:
You still left the extra spaces after the span class in all of these, and now there's an extra space inside of the span class. For example, line 268 should be `"span class='notice'>You shake the [src] trying to wake [t_him] up!</span>"`